app/scripts/controllers/main.js

```diff
@@ -57,18 +61,23 @@ angular.module('moodCatApp')
     $scope.selectRoom = function selectRoom(room) {
       $scope.activeRoom = room;
       $scope.loadSong(room.song.soundCloudId);
-    }
 
-    /** CHAT **/
+      //Fetch chat
```

Comment:
So, you would propose to change fetchMessages() to fetchChat(). Or shall I just delete that comment?